extra/nrnivmodl_core_makefile.in

```diff
@@ -30,6 +30,7 @@ _cm =,
 
 # We rebuild the include dirs since a lot of stuff changes place
 INCLUDES = $(INCFLAGS) -I$(incdir) -I$(incdir)/coreneuron/utils/randoms
+INCLUDES += -I@CORENEURON_PROJECT_SOURCE_DIR@/external/Random123/include
```

Comment:
will this continue working after `make install` ? I think we need to make sure that random123 is where we expect it to be and the generated makefile shoudl be able to find it when we run `nrnivmodl-core` from the installation target.

Reply:
I don't think we have any mod file in tests that use Random123. Would be good idea to add that. I can help.

Reply:
> will this continue working after make install ? 
As soon as you remove source, it won't work. I fixed this.